addon/components/offering-form.js

```diff
@@ -129,23 +67,25 @@ export default Component.extend(ValidationErrorDisplay, Validations, {
     }
 
     return defaultStartDate.toDate();
-  }),
-  durationHours: computed('startDate', 'endDate', function(){
-    const startDate = this.get('startDate');
-    const endDate = this.get('endDate');
+  }
+
+  @IsInt() @Gte(0)
+  get durationHours() {
+    const startDate = this.startDate;
+    const endDate = this.endDate;
 
     if (isEmpty(startDate) || isEmpty(endDate)) {
       return 1;
     }
     const mStart = moment(startDate);
     const mEnd = moment(endDate);
-    const diffInHours = mEnd.diff(mStart, 'hours');
+    return mEnd.diff(mStart, 'hours');
+  }
 
-    return diffInHours;
-  }),
-  durationMinutes: computed('startDate', 'endDate', function(){
-    const startDate = this.get('startDate');
-    const endDate = this.get('endDate');
+  @IsInt() @Gte(0) @Lte(59)
+  get durationMinutes() {
```

Comment:
This is fun! Sort of seems like maybe the future.

Reply:
yea, chaining them is the way to go.